packages/commonwealth/client/scripts/views/pages/Editor/useEditorErrorHandler.ts

```diff
@@ -0,0 +1,10 @@
+type ErrorPayload = Readonly<{
+  error: string;
+  source: string;
+}>;
+
+export function useEditorErrorHandler(): (err: ErrorPayload) => void {
```

Comment:
does it really needs to be a hook? looks like just a simple util function

Reply:
It's not a big deal but making it a hook means we can make it use a snackbar easily.  It's a placeholder for now.  This way it's always a hook and doesn't need to refactor the editor in the future.  

We can't re-render inside the MDXEditor or the cursor will be reset so I have to make sure all dependencies are memoized.